<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditCritereDialog</class>
 <widget class="QDialog" name="EditCritereDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>382</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>303</width>
    <height>382</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>303</width>
    <height>382</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_5">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Modification des parametres de listage&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="afficherTous">
       <property name="text">
        <string>Afficher tous</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="onlyMounth">
       <property name="text">
        <string>Durant un mois</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Mois: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mounth">
          <item>
           <property name="text">
            <string>Janvier</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fevrier</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mars</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Avril</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mai</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Juin</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Juillet</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Aout</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Septembre</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Octobre</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Novembre</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Decembre</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Année:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="yearMounth">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>2000</number>
          </property>
          <property name="maximum">
           <number>3000</number>
          </property>
          <property name="value">
           <number>2020</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="onlyYear">
       <property name="text">
        <string>Durant une année</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Année:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="year">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximum">
          <number>3000</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QRadioButton" name="twoDates">
       <property name="text">
        <string>Entre 2 dates</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Premiere date</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDateEdit" name="firstDate"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Seconde date</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateEdit" name="secondDate"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditCritereDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditCritereDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
